app/controllers/api/v1/allocations_controller.rb

```diff
@@ -71,6 +71,10 @@ def send_move_notifications
           Notifier.prepare_notifications(topic: move, action_name: 'create')
         end
       end
+
+      def includes
+        params.fetch(:include, nil)&.split(',') || AllocationSerializer::INCLUDED_ATTRIBUTES
```

Comment:
We don't require a `fetch` here as `params[:fetch]` will return `nil` if empty by default. Also this might be better suited in a service class, if we intend to further build handling errors and sparse fields and to allow this to be tested in isolation as well.